study.md

```diff
@@ -160,15 +168,19 @@ What is a response? What does it contain? What are some common status codes in a
 response and what do they mean?
 
 ```md
-<!-- your answer here -->
+A response is simply the server responding to the request from the client.
+
+Common codes that I have seen in my work:
+400-Bad request: This means that the server can't process a request because of a client error.
+401-Unauthorized: The client that made the request is not authorized to view the information.
 ```
 
 ## Make a curl request
 
 Using curl, how would you get the content from Reddit.com?
 
 ```md
-<!-- your answer here -->
+curl --request GET reddit.com
```

Comment:
I believe you will need the entire URL for this to work, the `http` states which protocol you are using to send the request.

`curl --request GET https://www.reddit.com`